CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [TBD]
+### Added
+* Regex support in `federated-meta-stores.mapped-databases`. [#59](https://github.com/HotelsDotCom/waggle-dance/issues/59).
```

Comment:
I think the README needs to be updated to indicate that the above now supports regex, not just a plain list. Can probable be very similar to what we say above for `primary-meta-store.writable-database-white-list`